### PR TITLE
Fix horizontal scroll

### PR DESCRIPTION
**Pull request by tom5079**
WIP

### CODE REVIEW

**Comment by tom5079** [comment]
done. resolves #3, resolves #33

File: recyclerviewfastscroller/src/main/java/com/qtalk/recyclerviewfastscroller/RecyclerViewFastScroller.kt [hunk 1]
```diff
@@ -343,23 +383,31 @@ class RecyclerViewFastScroller @JvmOverloads constructor(
...
+                            FastScrollDirection.VERTICAL ->
+                                motionEvent.rawY - yAbsPosition - handleOffset
+                        }
 
                         // move the handle only if fastScrolled, else leave the translation of the handle to the onScrolled method on the listener
 
                         if (isFastScrollEnabled) {
-                            moveViewByRelativeYInBounds(handleImageView, currentRelativeYPos)
-                            moveViewByRelativeYInBounds(
+                            moveViewByRelativeInBounds(handleImageView, currentRelativePos)
```
Comment (shahsurajk) [inline recyclerviewfastscroller/src/main/java/com/qtalk/recyclerviewfastscroller/RecyclerViewFastScroller.kt]: the name of this function seems a bit ambiguous, let change it to: 
```suggestion
                            moveViewByRelativeDirectionInBounds(handleImageView, currentRelativePos)
```

**Review by shahsurajk — CHANGES_REQUESTED** [review]
Looks really good @tom5079 . Just one small nit.

File: recyclerviewfastscroller/src/main/java/com/qtalk/recyclerviewfastscroller/RecyclerViewFastScroller.kt (hunk 1 above)
Comment (tom5079) [reply recyclerviewfastscroller/src/main/java/com/qtalk/recyclerviewfastscroller/RecyclerViewFastScroller.kt]: Alrighty 👍

Comment (tom5079) [reply recyclerviewfastscroller/src/main/java/com/qtalk/recyclerviewfastscroller/RecyclerViewFastScroller.kt]: @shahsurajk how about `moveViewToRelativePositionWithBounds`? What it really does is that it moves a view relative to its parent.

Comment (shahsurajk) [reply recyclerviewfastscroller/src/main/java/com/qtalk/recyclerviewfastscroller/RecyclerViewFastScroller.kt]: can do. works!

**Review by shahsurajk — APPROVED** [review]

**Comment by tom5079** [comment]
Great :)

**Pull request merged**